### ISSUE TITLE
MDCheckbox in OneLineAvatarIconListItem gets updated too frequently + suggested fix

### ISSUE THREAD

**Issue by mkc2019** [kivymd/KivyMD#258]
### Description of the Bug
If I place a MDCheckBox inside a OneLineAvatarIconListItem, when I press the check box, it gets checked and then unchecked immediately. Using logcat I realized that the system considered the button pressed twice, although I only tapped it once. 

### Code and Logs
My kv file:
```
MDList:
  id: alert_table
  padding: 0,0,0,0
  # Switches - HR alert
  OneLineAvatarIconListItem:
      text: "Display Heart Rate Alert"
      font_style: "H6"
      IconLeftWidget:
          icon: "heart-pulse"
      SwitchRightWidget:
          id: doHeartRateAlert
```
My .py file that defines SwitchRightWidget (it was named switch because i originally use MDSwitch)

```
class SwitchRightWidget(IRightBody, MDCheckbox):
    pass
```
### Screenshots
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/55228926/79850933-2f2f2100-83f7-11ea-9660-0fc15906e66d.gif)

### Suggestion
Fix it using min_state_time, as in the method used by kivy button.
```
from time import time

class SwitchRightWidget(IRightBody, MDCheckbox):
    def __init__(self,**kwargs):
        super().__init__(**kwargs)
        self.__touch_time = 0
        self.min_state_time = 0.1

    """The app is so sensitive that one tap is counted twice. Add an IF statement to prevent that."""
    def _do_press(self):
        if time() - self.__touch_time > self.min_state_time:
            self.__touch_time = time()
            super()._do_press()
```

### Versions
* OS: Android 10
* Python: 3.6.9
* Kivy: 1.11.1
* KivyMD: 0.104.0

**Comment by HeaTTheatR** [comment]
@mkc2019 Strange, but I can not reproduce this behavior.

![ezgif-4-1bc2f35f2575](https://user-images.githubusercontent.com/16930280/79854971-fbda9b00-83d2-11ea-849e-a4a0db190e71.gif)

**Comment by mkc2019** [comment]
This only happens in an android device. It works fine on PC. Did you do your test in an android device?

**Comment by HeaTTheatR** [comment]
@mkc2019 

![44444](https://user-images.githubusercontent.com/16930280/79856399-f8481380-83d4-11ea-8dd1-910280cdb826.gif)

**Comment by HeaTTheatR** [comment]
@mkc2019 Show your code that reproduces this behavior. We need to figure this out if such a problem exists.

**Comment by mkc2019** [comment]
@HeaTTheatR I would like to, but there is a problem. Since this is company property, i don't think i have the right to share the whole code with you... Let me check if i can write a short code to reproduce the problem tomorrow when i am back at office :)

**Comment by mkc2019** [comment]
...This is awkward. I tried building a simple app containing only OneLineAvatarIconListItem as well as Checkbox, but I cannot reproduce the problem either. So, either because:

1. This problem has been fixed in the master branch at some point (I may be using some old version of 0.104.0)
2. Somewhere in my code I messed up, but I don't know where and why.

In any case, since I know how to fix this in my code, it is not a big concern for me. Though I would still suggest you add a slight time delay for _do_press() to prevent checked Checkbox gets unchecked immediately afterwards. 
@HeaTTheatR  can safely close this issue.

[Update]:
Yep. I definitely messed up my code. 
In my main.py, there was a line saying:
`Config.set('input', 'mouse', 'mouse,multitouch_on_demand')`
Not sure why I put it there in the first place. Removing this line solves the problem.
